libnavui-maps/src/test/java/com/mapbox/navigation/ui/maps/internal/route/line/MapboxRouteLineUtilsTest.kt

```diff
@@ -1520,33 +1396,10 @@ class MapboxRouteLineUtilsTest {
         assertEquals("severe", trafficExpressionData[2].trafficCongestionIdentifier)
     }
 
-    @Test
-    fun getRouteLineTrafficExpressionDataWithNoRouteOptions() {
```

Comment:
`NavigationRoute`s require `RouteOptions` to be present so this test is irrelevant.